package.json

```diff
@@ -50,7 +50,7 @@
   "sideEffects": false,
   "main": "build/index.js",
   "scripts": {
-    "build": "babel --out-dir build src --ignore src/__tests__",
+    "build": "babel --out-dir build src --ignore src/__tests__ --source-maps --delete-dir-on-start --minified --no-comments",
```

Comment:
Do we also need a `NODE_ENV=production` ?

Reply:
I don't think so it, but I'll check.